pkgs/servers/mail/listmonk/default.nix

```
@@ -0,0 +1,54 @@
+{ lib, buildGoModule, fetchFromGitHub, callPackage, stuffbin }:
+let
+  frontend = callPackage ./frontend.nix { };
+  vfsMappings = [
+    "config.toml.sample"
+    "schema.sql"
+    "queries.sql"
+    "static/public:/public"
+    "static/email-templates"
+    "${frontend}:/admin"
+    "i18n:/i18n"
+  ];
+  putIntoBinary = mapping: ''
+    ${stuffbin}/bin/stuffbin -a stuff -in $out/bin/listmonk -out $out/bin/listmonk "${mapping}"
+  '';
+in
+buildGoModule rec {
+  pname = "listmonk";
+  version = import ./version.nix;
+
+  vendorSha256 = import ./vendor-hash.nix;
```

```suggestion
  version = "...";
```
Please fill that in instead of importing it.

---

Done.